setup.py

```diff
@@ -321,6 +321,7 @@ def run(self):
     "core": [
         PYQT_DEP,
```

Comment:
Would it be better to put it in the PYQT_DEP ? (maybe renaming this to `PYQT_DEPS`)

Reply:
25e3033

Reply:
:+1: